### DISCUSSION TITLE
Feedback to credential detection in user secrets scanning

### DISCUSSION THREAD

**Discussion by ecki** [community/community#112635] [Code Security] [answered]
### Select Topic Area

Product Feedback

### Body

This screenshot shows a false positive, where the pattern is a bit too eager it seems. Maybe you can require blank or quotes before the string, but not dot (but even then the risks of missdetection is high). Funny enough the screenshot also shows a „real“ secret - which was not detected.
 
![image](https://github.com/community/community/assets/361432/e6fb96c2-841a-4450-b840-477f0227db54)

**Comment by akibul123** [comment]
# fahim bot~

**Comment by aashah** [comment, marked as answer]
Hey @ecki,

Sorry for the late reply. Thanks for sharing this. It is indeed a false-positive, and a mistake we made when we started releasing a handful of features dedicated for open-source repositories (such as push protection + secret scanning for all).

We're working on a fix to just go ahead and delete these false-positives altogether.

Sorry about the spam! 😢 
Aakash
